tests/unit/core/reducers/test_addons.js

```diff
@@ -100,8 +100,22 @@ describe(__filename, () => {
     expect(state.byID[extension.id]).toEqual({
       ...extension,
       iconUrl: extension.icon_url,
-      installURLs: {
-        [OS_ALL]: 'https://a.m.o/files/321/addon.xpi',
+      platformFiles: {
+        [OS_ALL]: {
```

Comment:
This test data needs to inherit from a fake file object so that you don't have to repeat all fields of data. We have currently been using [`createFakeAddon().current_version.files[0]`](https://github.com/bobsilverberg/addons-frontend/blob/89f906bbbf0d51dd6f514e5df92746027820a1c3/tests/unit/amo/helpers.js#L299) as a base but maybe we need `createFakeFile()` for clarity.

Reply:
This was a pretty easy fix using `fakeAddon` in the test, and I don't really think we need a `createFakeFile()` currently, so I just went with the quick fix.